src/test/java/uk/gov/hmcts/ccd/integrations/IntegrationTest.java

```diff
@@ -18,7 +18,7 @@
 
 @RunWith(SpringRunner.class)
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
-@AutoConfigureWireMock(port = 0)
+@AutoConfigureWireMock(port = 12345)
```

Comment:
What's the reason for moving away from a random port? This will cause collisions ands test hanging. Could this be reverted?